client/utils/functions.js

```diff
@@ -0,0 +1,22 @@
+
+//my intent is to use this function in the homepage comp of app.jsx, I'd like to house it here to make my code cleaner
+//I want to learn how to possibly resue line 8? can the http be saved as a variable and I add diff endpoints based my server?
```

Comment:
This is a great idea.

Reply:
Thank you!  I think I have my functions.js and variables.js working.  I'll share in our next mentor meeting!